### DISCUSSION TITLE
Invalid Configuration Trying To Add Custom Domain

### DISCUSSION THREAD

**Discussion by rahulserver** [vercel/community#6921] [Help] [answered]
### Summary

I added a custom domain to my vercel project and updated the dns settings in the registrar to point to ns1.vercel-dns.com and ns2.vercel-dns.com
The change has propagated and can be confirmed from whois here: https://www.whois.com/whois/visageinfo.tech
But I get "invalid configuration" on my vercel custom domains page(check attached screenshot).

I have already waited for over 48 hours and still see no progress.
So what do I need to fix it?
<img width="1374" alt="Screenshot 2024-05-21 at 5 17 41 AM" src="https://github.com/vercel/community/assets/5101740/c32c7569-668d-407b-8097-4f79fb13678f">

### Example

visageinfo.tech

### Steps to Reproduce

Just add a custom domain to a project and then update its dns settings from the regisrar.

**Comment by sami5001** [comment]
I'm having the exact same issue for over a month now.

**Reply by amyegan** [reply, marked as answer]
The [domain troubleshooting guide](https://vercel.com/docs/projects/domains/troubleshooting) has some tips that should help you sort out common configuration issues related to custom domains. If you share the domains I can also take a closer look.

**Comment by nappalm** [comment]
How did you solve it? I also added my domain and everything was working correctly but then it started to fail, now it says that I have configured other dns that I have never set up.

```
verify1.registrar.eu
verify2.registrar.eu
verify3.registrar.eu
```

**Reply by amyegan** [reply]
I posted a response to your other post https://github.com/vercel/community/discussions/7168

**Reply by sami5001** [reply]
@nappalm I did all the troubleshooting steps and at the end found out that my TDL registration authority had put a ServerHold. If the logs from the command that @amyegan shared don't show this, contact your domain registrar.

**Comment by amyegan** [comment]
@rahulserver Your domain is showing valid configuration on Vercel now. Please let me know if you're still having trouble with it

**Reply by SergiuPlesco** [reply]
I am having the same issue. But there were no problems a couple of days ago. Vercel nameservers are added where needed. Is that a vercel issue or the provider with whom i bought the domain?

**Reply by amyegan** [reply]
You can use tips from the [domain troubleshooting guide](https://vercel.com/docs/projects/domains/troubleshooting) to figure out where the problem is happening. 

If `dig ns [domain]` returns an unexpected value, then you'll know something is wrong with the DNS configuration and you may need to contact your DNS provider for help. That could be a sign of a server hold as was the case in https://github.com/vercel/community/discussions/6921#discussioncomment-9752480

There are a few other common issues covered in the troubleshooting guide, so I recommend reading through it. You can check for current records using [`dig`](https://en.wikipedia.org/wiki/Dig_(command)) or an online tool such as [whatsmydns.net](https://www.whatsmydns.net/) to check global DNS propagation.

**Reply by SergiuPlesco** [reply]
Thanks for your answer.

[a comment by an automated account is not shown]
